src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeonardoModel.scala

```diff
@@ -23,7 +23,7 @@ import org.broadinstitute.dsde.workbench.model._
 import spray.json._
 
 // Create cluster API request
-case class ClusterRequest(labels: LabelMap = Map(),
+case class ClusterRequest(labels: Option[LabelMap] = Option(Map()),
```

Comment:
this was the only required field. making it optional allows me to use `ClusterRequest` for the PATCH payload and an empty `LabelMap` is already equivalent to leaving the field out